### ISSUE TITLE
subtitles does not show with slider

### ISSUE THREAD

**Issue by marathia** [yourhead/s3#531]
As per title: can't get a subtitle to show with a slider.

**Comment by yourhead** [comment]
not every control has a subtitle (the rule and link are on this list too) -- these are purposeful commissions where the addition of a subtitle seemed to be more often confusing than helpful.

i'd be open to well considered arguments to the contrary. however without any new evidence i think i'm going with my initial design goals and impressions -- which was to leave this out.

**Comment by marathia** [comment]
I do understand your reasoning, but for me it's also a matter of consistency. Number serves more or less the same function as a slider, but can have a subtitle.

Although not exactly evidence, here's an example: I'm working on a stack that can contain a lot of images that can be scaled down in edit mode to save space. My preferred option to achieve this would be a slider (from 1 to 8) for the number of images next to each other. To avoid confusion, I would like to add a subtitle emphasizing that this is for Edit mode only.

Alternatives are to put that text in a tooltip or an info; or use a number instead of a slider. IMHO those are all less appealing solutions.

**Comment by yourhead** [comment]
Yes, I understand all those things.

Please keep in mind that I have, obviously prototyped this already, tried it in the app, then took extra time to write code to make this control exceptional -- to make the subtitle property apply differently to this one.  This took me extra work and time that I was willing to spend because I felt so strongly that this label was inappropriate.

Simply telling em you have a single instance where it might be helpful is not sufficient.  I'm sorry.

The challenge here is that all the subtitles appear underneath the control.  In the cases where I've opted to remove it wasn't because there was no use for a subtitle -- a subtitle would **always** be helpful.  The reason for removing is that putting text in that spot created some problem.

In this case it has to do with the tick marks.  Currently if you add tick marks they are labeled with the min/max values of the slider as is normal.  The label needs to squeeze in between these tick marks, but somehow be at a different enough position that it's clear the label is not a middle-tick-mark-label but rather a subtle for the whole control.

Vertically I could not find a position that made that clear that didn't also create an uncomfortable white space below the control.

Horizontally balancing the control between the tick marks is also challenging a problematic.  Should the label be centered in the middle of the control -- or between the bounds of the min/max labels?  If one of the labels is very large (10000000) -- then the simply centering the subtitle looks unbalanced and runs the risk over overlapping those other labels.

In short:  functionally a label is always useful -- however aesthetically there are times when adding more text is just plain ugly -- or worse confusing.  To me this seemed like a very clear instance of that later.

In order to reverse my judgement and get me to rewrite all of that code you'd have to show me a similar control used well in another app that solved some/all of these problems that seemed insurmountable to me.
